lib/configuration.rb

```diff
@@ -53,6 +53,14 @@ def agents_install_dir
       Pathname.new(env('AGENTS_INSTALL_DIR', '.')) + 'go-agents'
     end
 
+    def include_plugin
```

Comment:
LGTM, mostly. Make this return a boolean instead and call it `def include_plugin?`